features/old/api/metrics/creation.feature

```diff
@@ -7,6 +7,7 @@ Feature: Metric creation
     Given a provider "foo.example.com"
     And an application plan "Basic" of provider "foo.example.com"
     When current domain is the admin domain of provider "foo.example.com"
+    And all the rolling updates features are off
```

Comment:
Seeing this made me think: shouldn't be off by default?
So I've checked myself:
![image](https://user-images.githubusercontent.com/11318903/63159390-c26bb780-c01b-11e9-9a54-4a9d8b0a3608.png)
![image](https://user-images.githubusercontent.com/11318903/63159406-cac3f280-c01b-11e9-9f17-de9d4f45a10b.png)

So if this was added for 'this is the expected behaviour for when the rolling updates are off', then great, but if it was added because otherwise it fails depending on the order of the tests, then we should fix that (but out of scope of this PR)